components/navbar/sidebar.snip.html

```diff
@@ -17,11 +17,14 @@
 }}
 
 <!-- Start Sidebar -->
-<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}}" layout="nodisplay">
+<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} {{sidebarclasses}}" layout="nodisplay">
```

Comment:
one small nit 
can you do 
```json
sidebar: {
  "classes": "general purpose classes",
  "layout-classes": "ablsolute, relative, float etc",
  "spacing-classes": "p0, m0, mx-auto etc",
  "resposive-display-classes": "sm-hide md-hide",
  "resposive-layout-classes": "md-col12, sm-col6 etc"

}
```

I think this is becoming a pattern now ,  when we build the next version of ampstart , it will help separate markup and layout very easily.